### DISCUSSION TITLE
More drift with higher resolution?

### DISCUSSION THREAD

**Discussion by msoiron** [stella-cv/stella_vslam#405] [Q&A] [answered]
Hi all,

We are using stella vslam in monocular mode with a Theta X 360° camera. For some videos we are seeing a lot of drift in the SLAM result. The strange thing is that this drift is actually larger when we use the native resolution of the video (3840 x 1920 pixels) instead of a reduced version (of about 30%).

Is there an explanation for that or are we maybe doing something wrong?

**Comment by ymd-stella** [comment, marked as answer]
Unnecessarily high resolution will degrade performance. There is nothing good about increasing the resolution.
